views/partials/subpages/confirm.hbs

```diff
@@ -1,3 +1,3 @@
 <span class="dashboard-email-sent bold txt-purple7 email-confirmed txt-cntr mw-550"> {{ getString "email-verified" }}</span>
-<span class="txt-purple7 email-confirmed txt-cntr mw-550">{{{ makeEmailAddedToSubscriptionString email }}}</span>
+<span class="txt-purple7 email-confirmed txt-cntr mw-550 overflow-break">{{{ makeEmailAddedToSubscriptionString email }}}</span>
 <span class="txt-purple7 email-confirmed txt-cntr mw-550">{{{ makeEmailVerifiedString }}}</span>
```

Comment:
Only UGC I saw in these strings is the email address. I tried adding an email address of `<script>alert("XSS");</script>` to see if I could get that stored in the DB, and then send the `verify?token=` link to a different browser, but we have server-side validation on that form field, so it looks 👌